packaging/container/Dockerfile

```diff
@@ -1,35 +1,36 @@
-FROM quay.io/centos/centos:stream8 as builder
+FROM quay.io/centos/centos:stream9 as builder
 
-RUN dnf -y update && dnf install -y golang make python3 python3-pip python3-wheel git
+RUN dnf -y update && dnf install -y golang make python3 python3-pip git
+RUN pip install wheel
 
 ADD source.tar.gz /source
 WORKDIR /source
 RUN make
 
-FROM quay.io/centos/centos:stream8
+FROM quay.io/centos/centos:stream9
 ARG VERSION
 
 LABEL license="ASL2"
 LABEL name="receptor"
 LABEL vendor="ansible"
 LABEL version="${VERSION}"
 
-RUN dnf -y update && \
-    dnf -y install epel-release && \
-    dnf -y install tini python3-pip python3-wheel && \
-    dnf clean all
-
 COPY receptorctl-${VERSION}-py3-none-any.whl /tmp
 COPY receptor_python_worker-${VERSION}-py3-none-any.whl /tmp
-RUN pip3 install /tmp/*.whl
-RUN rm /tmp/*.whl
-
 COPY receptor.conf /etc/receptor/receptor.conf
+
+RUN dnf -y update && \
+    dnf -y install python3-pip && \
+    dnf clean all && \
+    pip install --no-cache-dir wheel dumb-init && \
```

Comment:
Note that I removed `tini` in favor of `dumb-init`.

- `epel-release` doesn't seem to exist in stream9 yet.
- dumb-init is available via pip
- dumb-init is the more active project
- dumb-init is already used by every Execution Environment.